### PR TITLE
Update wanda.py

### PR DESCRIPTION
**Pull request by henry-richard**
Test

### CODE REVIEW

**Comment by henry-richard** [comment]
Test

File: src/wanda.py [hunk 1]
```diff
@@ -16,4 +16,5 @@
 print("Wanda testing preflight 0.8.1")
 print("Wanda testing preflight 0.8.1")
 print("Wanda testing preflight 0.8.1")
+print("Wanda testing preflight 0.8.1")
```
Comment (wanda-carlson) [inline src/wanda.py:19]:
```suggestion
print("Wanda testing preflight 0.8.1")
print("Wanda testing preflight 0.8.1")
```

**Review by wanda-carlson — CHANGES_REQUESTED** [review]

**Review by wanda-carlson — APPROVED** [review]

**Review by wanda-carlson — APPROVED** [review]

**Comment by henry-richard** [comment]
Sure

**Comment by henry-richard** [comment]
@wanda-carlson

**Comment by henry-richard** [comment]
@wanda-carlson

**Pull request merged**